build/steps-init.yml

```diff
@@ -16,8 +16,8 @@ steps:
 
 
 ##
-# Bootstrap
+# Custom pre-reqs
 ##
-- pwsh: ./bootstrap.ps1
```

Comment:
The purpose of bootstrap scripts is to prep the repository for local development, including building some of the components, which was a mess to re-use in CIs. So I introduced new prereq/build/test scrips for native simulator but I didn't touch any of the bootstrap logic.